paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -316,9 +316,8 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
         dst_memory_p = handler.AcquireDstMemoryFromResidualDataMemory(
             user_residual_memory_p, to_void_cast<T>(output_data), pipeline);
       } else {
-        auto output_data = output->mutable_data<T>(
-            ctx.GetPlace(), residual_param->memory_size());
-        framework::TensorCopy(*residual_param, residual_param->place(), output);
+        output->ShareDataWith(*residual_param);
```

Comment:
The reasons for using the ShareDataWith function and the issue link (https://github.com/PaddlePaddle/Paddle/issues/22964) can be added here.

Reply:
Ok, I'll do that in a moment